dds_web/api/files.py

```diff
@@ -311,6 +311,8 @@ def get(self):
             subpath = "."
             if flask.request.args.get("subpath"):
                 subpath = flask.request.args.get("subpath").rstrip(os.sep)
+                if subpath is "":  # in case the subpath is empty after rstrip
+                    subpath = "."
```

Comment:
Could we de-duplicate between the 2 functions? I know that they implement the 2 different API versions but, given that we don't have a firm timeline for the deprecation of the older, it might be more maintainable to have them more de-duplicated.

Reply:
However, the subpath is obtained through two different ways in the two methods:
On one is extra_args.get("subpath") on the other flask.request.args.get("subpath")

path vs query paramter